kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -35,7 +35,7 @@ def subsample_fourier(x, k):
     return res
 
 
-def pad_1d(x, pad_left, pad_right, mode='constant', value=0.):
+def pad(x, pad_left, pad_right, mode='reflect', value=0.):
```

Comment:
should the defaults for `pad_left`, `pad_right` be `=0`?

Reply:
If they were zero before, let's leave them at zero now.